src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -1876,6 +1876,15 @@ File createTempFile(String prefix, String suffix) throws IOException {
             if (workspaceTmp.getAbsolutePath().matches(".*[ ()|?*].*")) {
                 return createTempFileInSystemDir(prefix, suffix);
             }
+            /* JENKINS-62579 - Windows git fails its call to GIT_SSH and GIT_ASKPASS if the
```

Comment:
Is this because of an issue in Jenkins or in git?

Reply:
I assume it is an issue with git for Windows, since git for Linux works correctly in the same conditions.  I did not spend the time to research the details of the failure.  Git client plugin versions that only used the system temporary directory were able to meet the user needs even with Cyrillic characters in the job name.  This restores that behavior and includes other non ISO-8859-1 project names in the same exclusion.